src/set_theory/cardinal/basic.lean

```diff
@@ -621,6 +621,14 @@ theorem bdd_above_iff_small (s : set cardinal.{u}) : bdd_above s ↔ small.{u} s
   { simp_rw [subtype.val_eq_coe, equiv.symm_apply_apply], refl }
 end⟩
 
+@[simp] lemma lift_mk_shrink (α : Type u) [small.{v} α] :
+  cardinal.lift.{max u w} (# (shrink.{v} α)) = cardinal.lift.{max v w} (# α) :=
+lift_mk_eq.2 ⟨(equiv_shrink α).symm⟩
+
+@[simp] lemma lift_mk_shrink' (α : Type u) [small.{v} α] :
+  cardinal.lift.{u} (# (shrink.{v} α)) = cardinal.lift.{v} (# α) :=
+lift_mk_eq'.2 ⟨(equiv_shrink α).symm⟩
```

Comment:
```suggestion
lift_mk_shrink.{u v 0} α
```
If this is true, this gives the human reader a better understanding of the connection of the two lemmas.

Do we also want a version without `cardinal.lift` on the right? That is, with `(α : Type (max u v)) [small.{v} α]`?

Reply:
That is true, and I've provided that other version.